test/compress/parameters.js

```diff
@@ -1,4 +1,7 @@
 arrow_functions: {
+    options = {
+        arrows: true,
```

Comment:
Why is the new `arrows` option now required for this test? There are no AST_Function nodes present.